veps/sig-storage/incremental-backup.md

```diff
@@ -291,16 +292,22 @@ Optional. Required in `push` mode. Specifies the name of the PVC where the backu
 Optional. If set to true, the VM's filesystem will not be quiesced before the backup. By default, the system attempts to quiesce the filesystem to ensure data consistency.
 - `forceFullBackup` <br>
 Optional. If a backupTracker is provided and this field is set to true, force a full backup (instead of incremental using `latestCheckpoint`). The `latestCheckpoint` will be updated with the new full backup checkpoint.
+- `tokenSecretRef`<br>
+Optional. Required in `pull` mode. Specifies the name of the Secret containing the authentication token for the backup export. 
+- `ttl`<br>
+Optional. Only relevant for `pull` mode. Specifies the time to live for the backup, defaults to 2 hours if not specified.
 
 **Status:**
 - `checkpointName`<br>
 The name of the checkpoint created for the current backup. This field is updated by the backup controller.
 - `conditions`<br>
-Represents the current state of the backup process, such as `Initializing`, `Done`, `Failed` or `Deleting`
+Represents the current state of the backup process, such as `Initializing`, `Progressing`, `Ready`, `Done`, `Failed`, or `Deleting`.
 - `backupType` <br>
 Either `Full` or `Incremental`. Indicates the backup's scope, allowing users to confirm it aligns with their expectations or understand any deviation.
+- `includedVolumes` <br>
+A list of the names of the volumes that were included for the backup.
```

Comment:
I believe its better to also include the endpoints from which the users can pull the disks over. I see in the explanations below that you intended to have the user get this info from vmexport. I think we would prefer to have all the relevant info for backup in the same crd of the virtualmachinebackup instead of refering the user to get another CR of the export and get the data from there.. I know its basically copy the info that is in the vmexport status to the vmbackup status but I think the vmexport is somewhat of and implementation detail on how we expose the endpoints and dont want to bother the user with extra info it doesnt need to know exists. WDYT?